lib/dangermattic/plugins/common/git_utils.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+module Danger
+  # Represents a utility plugin for working with Git: to check added and modified lines in Git diffs,
+  # to determine the type of change (added, removed, or other) in a diff line, and to retrieve lists of
+  # added, modified, and deleted files.
+  #
+  # @example Check if there is a "TODO" in Ruby files:
+  #          git_utils.check_added_diff_lines(
+  #            file_selector: ->(path) { path.end_with?('.rb') },
+  #            line_matcher: ->(line) { line.include?('TODO') },
+  #            message: 'Found a TODO in a Ruby file'
+  #          )
+  #
+  # @example Get added lines from a diff patch:
+  #          added_lines = added_lines(diff_patch: diff_patch)
```

Comment:
Should this and the other examples call `git_utils`, too?

```suggestion
  #          added_lines = git_utils.added_lines(diff_patch: diff_patch)
```

Reply:
Yes! Added here: 0121c04c3d2e9eb57fc8f082806a07a2eb30e728